applications/FluidDynamicsApplication/custom_processes/flows_measuring_process.h

```diff
@@ -0,0 +1,280 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Miguel Angel Celigueta
+//
+//
+
+#ifndef KRATOS_FLOWS_MEASURING_PROCESS_H
+#define KRATOS_FLOWS_MEASURING_PROCESS_H
+
+// System includes
+#include <string>
+#include <iostream>
+
+// External includes
+
+// Project includes
+#include "processes/process.h"
+#include "includes/define.h"
+#include "includes/model_part.h"
+#include "includes/cfd_variables.h"
+#include "utilities/openmp_utils.h"
+
+// Application includes
+
+
+namespace Kratos
+{
+///@addtogroup FluidDynamicsApplication
+///@{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+class FlowsMeasuringProcess : public Process
+{
+public:
+    ///@name Type Definitions
+    ///@{
+
+    /// Pointer definition of FlowsMeasuringProcess
+    KRATOS_CLASS_POINTER_DEFINITION(FlowsMeasuringProcess);
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /// Constructor.
+    FlowsMeasuringProcess(Model& rModel, Kratos::Parameters::Pointer pParameters) : mrModel(rModel)
```

Comment:
```suggestion
    FlowsMeasuringProcess(Model& rModel, Parameters pParameters) : mrModel(rModel)
```

Parameters should be passed by Value due to internal memory management